modules/xss.py

```diff
@@ -146,7 +146,7 @@ def xss_get_url(url,method,headers,body,scanid=None):
                 dbupdate.insert_record(attack_result)
                 result = True
 
-        xss_request_uri = req.api_request(parsed_url+'/?test='+payload,"GET",headers)             
+        xss_request_url = req.api_request(parsed_url+'/?test='+payload,"GET",headers)             
         if xss_request_url.text.find(decoded_payload) != -1:
             impact = check_xss_impact(xss_request_uri.headers)
```

Comment:
should be here `xss_request_url` instead of `xss_request_uri`?

Reply:
@jc89 It should be xss_request_uri in line no 150.

Reply:
Closing this pull request. I have done a couple of changes in the module. Thanks!